front/src/dashboard/components/BSDList/BSDD/WorkflowAction/ProcessedInfo.tsx

```diff
@@ -95,6 +95,9 @@ function ProcessedInfo({ form, close }: { form: TdForm; close: () => void }) {
   ]);
 
   const TODAY = new Date();
+  const isFRCompany = nextDestination?.company?.siret ? true : false;
```

Comment:
Éventuellement comme ça ?

```suggestion
  const isFRCompany = Boolean(nextDestination?.company?.siret);
```